app/components/list-uploaded-assets/ListUploadedAssets.tsx

```
@@ -0,0 +1,89 @@
+'use client';
+import { fetchAllAssets } from '@app/api/livepeer/actions';
+import { Asset } from '@app/lib/types';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)